README.md

```diff
@@ -1,4 +1,4 @@
-<div align="center"><img src="https://github.com/aagarwal1012/Animated-Text-Kit/blob/master/display/cover.gif?raw=true"/></div>  
+<div align="center"><img src="display/cover.gif?raw=true"/></div>
```

Comment:
@awhitford, these relative paths actually don't work on https://pub.dev/packages/animated_text_kit, please revert these changes and create a new release.

Reply:
Ugh!  I was not thinking of `pub.dev` -- I made the change because from GitHub's perspective, it makes validating a branch more accurate -- otherwise it can get confusing when you make a change on a branch, but this still points to master. However, I get that `pub.dev` is more important. I'm sorry.